app/src/common/shared/com/igalia/wolvic/ui/widgets/WindowWidget.java

```diff
@@ -251,7 +251,6 @@ protected void initializeWidgetPlacement(WidgetPlacement aPlacement) {
         aPlacement.density = getBrowserDensity();
         aPlacement.visible = true;
         aPlacement.cylinder = true;
-        aPlacement.textureScale = 1.0f;
```

Comment:
We need to fix the pointer when we don't hard-code the textureScale as 1

Reply:
Fixed by https://github.com/Igalia/wolvic/pull/1086/files#diff-343d2af88acc48b1395827803aff28623f8942e6b7e87c7ea0598db849149a58R1097-R1100